src/pages/experiments/[experimentId]/data-exploration/components/heatmap/HeatmapSettings.jsx

```diff
@@ -45,11 +45,9 @@ const HeatmapSettings = () => {
     </Dropdown>
   );
 };
-
 HeatmapSettings.defaultProps = {
 };
-
 HeatmapSettings.propTypes = {
+  experimentId: PropTypes.string.isRequired,
```

Comment:
you are adding `experimentId` here as a required property, but I can't see where this prop is being used, can you point me to it?

Reply:
It was a settings I used in the previous implementation. I guess it got carried over when I created this branch. Deleted in commit c5eba06.